### PR TITLE
[ New Extension ] - Morning Coffee

### PR DESCRIPTION
**Pull request by oliverjanssen**
## Description

Morning Coffee is your Raycast extension with which you can easily organize routine web readings. Organize the websites you want to open simultaneously in your default browser and set a hotkey for launching 'Open Websites'.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi 👋 

Sounds a lot like [xecutor](https://www.raycast.com/GastroGeek/xecutor) functionality wise, have you tried that?

**Comment by olimynameis** [comment]
Hi @pernielsentikaer,

oh, sorry – I did not find that solution.

“xecutor” offers a bit more with apps, but “Morning Coffee” could be used with hotkeys (open websites, the real feature), and I prefer my approach of organizing websites by using Raycasts list component.

So shall I take back my pull request? Or is the store “open for competition”? Or is the hotkey-feature convincing? ;)

File: extensions/morning-coffee/src/components/forms/website-upsert.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      }
+    },
+    [onCreate, pop]
+  );
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm title={onCreate ? "Edit Website" : "Add Website"} onSubmit={handleSubmit} />
```
Comment (pernielsentikaer) [inline extensions/morning-coffee/src/components/forms/website-upsert.tsx:46]:
```suggestion
          <Action.SubmitForm title={onCreate ? "Add Website" : "Edit Website"} onSubmit={handleSubmit} />
```
Could you also add an icon?

File: extensions/morning-coffee/src/components/actions/website-delete.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Action, Icon } from "@raycast/api";
+
+interface ActionWebsiteDeleteInterface {
+  onDelete(): void;
+}
+
+export const ActionWebsiteDelete = ({ onDelete }: ActionWebsiteDeleteInterface) => {
+  return (
+    <Action icon={Icon.Trash} title="Delete Website" shortcut={{ modifiers: ["ctrl"], key: "x" }} onAction={onDelete} />
+  );
```
Comment (pernielsentikaer) [inline extensions/morning-coffee/src/components/actions/website-delete.tsx:10]:
```suggestion
return (
    <Action
      icon={Icon.Trash}
      title="Delete Website"
      style={Action.Style.Destructive}
      shortcut={{ modifiers: ["ctrl"], key: "x" }}
      onAction={onDelete}
    />
  );
```
Maybe also add a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert)

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 😊


I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/morning-coffee/src/components/forms/website-upsert.tsx (hunk 1 above)
Comment (olimynameis) [reply extensions/morning-coffee/src/components/forms/website-upsert.tsx]: Thanks, fixed and optimized it with icons.

File: extensions/morning-coffee/src/components/actions/website-delete.tsx (hunk 2 above)
Comment (olimynameis) [reply extensions/morning-coffee/src/components/actions/website-delete.tsx]: Added destructive styles, looks great. It also uses "confirmAlert" now.

**Comment by olimynameis** [comment]
Great feedback, added it.

**Review by pernielsentikaer — APPROVED** [review]
Hello 👋

I gave it a quick spin, works as expected, and is approved 🔥

**Pull request merged**